networking/eth2/src/testFixtures/java/tech/pegasys/teku/networking/eth2/Eth2NetworkFactory.java

```diff
@@ -89,7 +94,10 @@ public void stopAll() {
     protected AsyncRunner asyncRunner;
     protected EventBus eventBus;
     protected RecentChainData recentChainData;
-    protected GossipedAttestationConsumer gossipedAttestationConsumer;
+    protected GossipedOperationConsumer<ValidateableAttestation> gossipedAttestationConsumer;
+    protected GossipedOperationConsumer<AttesterSlashing> gossipedAttesterSlashingConsumer;
+    protected GossipedOperationConsumer<ProposerSlashing> gossipedProposerSlashingConsumer;
+    protected GossipedOperationConsumer<SignedVoluntaryExit> gossipedVoluntaryExitConsumer;
```

Comment:
We should set these to a no-op value rather than leaving them null.

Reply:
Done.